.github/workflows/cypress-Run-tests-with-npm-packages.yml

```diff
@@ -1,19 +1,31 @@
 # This is a sample workflow for Cypress.
-# It setups the environment and uses a cache for better performance on node and Cypress.
-# This action is based on the Kitchen Sink project (https://github.com/cypress-io/cypress-example-kitchensink/)
-# It can run tests by Browser and test file selection.
+# It setups the environment.
+# It run tests by Browser and test file selection.
+# It sends the tests to record on Cypress Dashboard if the user specifies it, if it is a pull request or a scheduled workflow run.
 
-name: Run Cypress tests without cache
+name: Run Cypress tests with npm packages
 
 on: 
-  - push
-  - workflow_dispatch
+  push:
+  schedule:
+    # runs at the minute 0 of the first hour of every monday
```

Comment:
I'd suggest to use a more human-oriented expression. 
Like: "Every monday at 1AM"? 😄

Reply:
Good point